source/partials/next_release/_release.md.erb

```diff
@@ -1,87 +1,88 @@
 
-    **Package plugins**
+<h4>Breaking Changes</h4>
 
-    * [Maven (Nexus) repository poller](https://github.com/aresok/go-maven-poller)
-    * [Docker Registry Poller - 1](https://github.com/hammerdr/go-docker-registry-poller)
-    * [Docker Registry Poller - 2](https://github.com/decoomanj/gocd-docker-material-poller)
-    * [Puppet Forge repository poller](https://github.com/drrb/go-puppet-forge-poller)
-    * [Debian repository poller](https://github.com/gocd-contrib/deb-repo-poller)
-    * [Cloud Foundry poller](https://github.com/Sounie/springer-gocd-cloudfoundry-plugin/blob/master/README.md)
+* <%= link_to_commit 'bd64d11d6d602ca4b0ab2318a3d504d07b472346', 'Removed Templates API v2', 3441 %>
 
-    **Task plugins**
+  Version 2 of the <%= link_to_api 'template-config', 'Templates API' %> was deprecated in 17.1. Version 3 of the API is available, and users are encouraged to use it.
+
+<h4>Deprecations</h4>
+
+Features that become superseded or irrelevant become *deprecated*, and are scheduled to be removed in some timeframe.
 
-    * [MSBuild Task](https://github.com/akanyer/gocd-msbuild-taskplugin/blob/master/README.md)
-    * [XUnit Converter](https://github.com/gocd-contrib/xunit-converter-task)
-    * [Publish to S3](https://github.com/indix-eng/gocd-s3-artifacts)
-    * [Fetch from S3](https://github.com/indix-eng/gocd-s3-artifacts)
-    * [Powershell Task](https://github.com/manojlds/gocd-powershell-runner/blob/master/README.md)
-    * [RapidDeploy Package Builder](https://github.com/MidVision/go-rapiddeploy/wiki#rapiddeploy-package-repository-plugin)
-    * [RapidDeploy Job Runner](https://github.com/MidVision/go-rapiddeploy/wiki#rapiddeploy-job-runner-task-plugin)
-    * [Maven Task Plugin](https://github.com/ruckc/gocd-maven-plugin)
-    * [Publish to Artifactory](https://github.com/tusharm/go-artifactory-plugin)
+The following features have been deprecated. Please note the timeframe of when they will be removed.
 
+* As of release 17.3, version 1 of the <%= link_to_api 'environment-config', 'Environment Config API' %> was deprecated. This version of the API will be removed in 17.6 release (scheduled to be released in June 2017). Version 2 of the API is available, and users are encouraged to use it.
 
-#### Contributors
+<h4>Contributors</h4>
 
 <%= [
-  'Alison Polton-Simon',
-  'Aravind SV',
-  'Bhupendrakumar Piprava',
-  'Ganesh S Patil',
-  'Jyoti Singh',
-  'Ketan Padegaonkar',
-  'Mahesh Panchaksharaiah',
-  'Markandan R',
-  'Naveen Bhaskar',
-  'Padma Mullagiri',
-  'Rajiesh Narayanan',
-  'Sallie Walecka',
-  'Sumanth Kumar Mora',
-  'Tomasz Setkowski',
-  'Varsha Varadarajan',
-  'Zabil Cheriya Maliackal',
+  "Ankit Srivastava",
+  "Aravind SV",
+  "Bhupendrakumar Piprava",
+  "Chris Northwood",
+  "Isabelle Carter",
```

Comment:
Add the following committers (for the website and blog): Huimin Li, David Rice, Badri J, Louda Peña, Jen Marley, Suzie Prince, Emily Luke